examples/with-multiple-theme-providers/README.md

```diff
@@ -0,0 +1,7 @@
+# Blade with `styled-components`
+
+When using Blade along with multiple theme providers (for example with `blade-old` or your own custom theming) you should namespace your own theme so it doesn't conflict with blade's theme tokens.
```

Comment:
Should we add some more details here like what issue it causes and how it merges both themes and how adding a namespace would solve the issue? Maybe attach a link to some `styled-components` issue that talks about the same?>

Reply:
Added in https://github.com/razorpay/blade/pull/889/commits/2652f7bbf35f8192b5f786b8ba8d5e8a468c34e7